recipes/gromacs/2024/gh200-mpi/environments.yaml

```diff
@@ -0,0 +1,26 @@
+gromacs:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  unify: when_possible
+  specs:
+  - cmake
+  - cuda@12
+  - gromacs@2024.1 +cuda +mpi +hwloc
+  - blas
+  - lapack
+  - hwloc
+  - fftw
+  - python@3.10
+  mpi:
+    spec: cray-mpich
+    gpu: cuda
+  variants:
+  - +mpi
+  - +cuda
+  - cuda_arch=90
+  views:
+    gromacs:
```

Comment:
See the packaging guide docs on how to create views here:
https://eth-cscs.github.io/alps-uenv/pkg-application-tutorial/#environments
Specifically, try to use "roots" to ensure that your view contains only what the users need.

Reply:
Yes. That is the idea. Once GROMACS works, then I will test out with PLUMED and VMD. And I will also try to make a custom installation of GROMACS on the `develop` view.